Tasks/AzureRmWebAppDeployment/task.json

```diff
@@ -475,6 +475,10 @@
         "ScriptStatusTimeout": "Unable to fetch script status due to timeout.",
         "PollingForFileTimeOut": "Unable to fetch script status due to timeout. You can increase the timeout limit by setting 'appservicedeploy.retrytimeout' variable to number of minutes required.",
         "InvalidPollOption": "Invalid polling option provided: %s.",
-        "MissingWebConfigParameters": "Some web.config parameters are missing"
+        "MissingAppTypeWebConfigParameters": "-appType attribute is missing in Web.config parameters.",
```

Comment:
Attribute '-appType' is missing in the Web.config parameters. Valid values for '-appType' are: 'python_Bottle', 'python_Django', 'python_Flask' and 'node'. <br/>
For example, '-appType python_Bottle' (sans-quotes) in case of Python Bottle framework.